bin/init.js

```diff
@@ -0,0 +1,16 @@
+const MongoClient = require('mongodb').MongoClient;
+const url = process.env.TALK_MONGO_URL;
```

Comment:
I'd rather see a graceful fallback like:

```js
const url = process.env.TALK_MONGO_URL || 'localhost';
```

And remove the error throwing below.